contribute/CONTRIBUTING-TO-K8S-DEMO.md

```diff
@@ -16,17 +16,18 @@ At a very high level, the process to contribute and improve is pretty simple:
 - Submit your Pull Request
 - Submit an issue to update the [User Documentation](https://github.com/openebs/openebs/blob/master/documentation/source/install/install_usecases.rst)
```

Comment:
Remove the link to User Documentation, this link is dead.

Reply:
I could give the link for the actual documentation at openebs-docs